src/Skoruba.IdentityServer4.Admin/Controllers/IdentityController.cs

```diff
@@ -155,7 +167,18 @@ public async Task<IActionResult> UserProfile(TUserDto user)
                 userId = userData.userId;
             }
 
-            SuccessNotification(string.Format(_localizer["SuccessCreateUser"], user.UserName), _localizer["SuccessTitle"]);
+            if ("InviteUser".Equals(submit))
```

Comment:
If the "Invite User" button was clicked, the invitation token claim is generated and the e-mail is sent out.  Notification of the "Success" on invitation sent will be raised to the view and shown as a green dialogue like other notifications.